deployment/GI01SUMO_Deploy.csv

```diff
@@ -135,72 +135,72 @@ AR-07-01,,AR21,,GI01SUMO-RII11-02-PCO2WC052,3,1,2016-07-10T17:44:00,2017-08-14T0
+AR21,,AR30-03,,GI01SUMO-SBD11-00-DCLENG000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGCON-BDCL11-70009,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-01-MOPAK0000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-MOPAK0-11351,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-03-HYDGN0000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-HYDGN0-42324,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-04-DOSTAD000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-DOSTAD-00495,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-05-SPKIRB000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-SPKIRB-00301,59.9449,-39.5751,,-6,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-06-METBKA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-METLGR-00019,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD11-08-NUTNRB000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-NUTNRB-00271,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-00-DCLENG000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGCON-BDCL12-70009,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-01-OPTAAD000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-OPTAAD-00165,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-02-FLORTD000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-FLORTD-01222,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-03-HYDGN0000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-HYDGN0-42509,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-04-PCO2AA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-PCO2AA-35261,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-05-WAVSSA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-WAVSSA-06431,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-06-METBKA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-METLGR-00021,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-08-FDCHPA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-FDCHPA-64402,59.9449,-39.5751,,-6,2659,buoy disappeared 10/12/17 - not recovered

```

Comment:
Deployment 4 stopDateTime for all components should be 2017-10-12T09:07:00 (the time the buoy went adrift and the riser fell to the seafloor).  
CUID for Buoy components should be "UNRECOVERED".

Reply:
fixed recovery cruise ID and stop date/time for deployment 4